students/CCSt130/lesson03/slicing_lab_ex3-1.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+"""
+Lesson03 :: Slicing Lab
+Exercise 3.1
+@author: Chuck Stevens :: CCSt130
+Created on Tue May 28 16:26:02 2019
+"""
+
+import math # for rounding
+import random # PRNG for fun
+
+
+test_str1 = ("The greatness of a man is not in how much wealth he acquires, but in his integrity and his ability to affect those around him positively...Bob Marley")
+
+test_str2 = ('There is no charm equal to tenderness of heart...Jane Austen')
+
+t_strings = (("All that we see or seem is but a dream within a dream...Edgar Allan Poe"),\
+                ("Lord, make me an instrument of thy peace. Where there is hatred, let me sow love...Francis of Assisi"),\
+                ("The only journey is the one within...Rainer Maria Rilke"),\
+                ("Good judgment comes from experience, and a lot of that comes from bad judgment...Will Rogers"),\
+                ("Think in the morning. Act in the noon. Eat in the evening. Sleep in the night...William Blake"),\
+                ("Life without love is like a tree without blossoms or fruit...Khalil Gibran"),\
+                ("No act of kindness, no matter how small, is ever wasted...Aesop"),\
+                ("Love cures people--both the ones who give it and the ones who receive it...Karl A. Menninger"),\
+                ("Dance like no one is watching...Satchel Paige"))
+
+def first_is_last(my_str):
+    """ Manipulate strings in various ways, such as swapping the first and last element. """
+    # Exchange first and last char in str
+    var_swap = my_str[-1:] + my_str[1:-1] + my_str[:+1]
+   
+    return(var_swap)
+
+def every_other(my_str):    
+    # Print str with every other letter removed
+    
+    new_str = my_str[::2]
```

Comment:
you can just return new seq directly -> `return seq[::2]`

`my_str` is a poor choice for var name here - your functions should support any sequence: tuple, list, or a string